assets/schema/ensemble_schema.json

```diff
@@ -1766,6 +1766,14 @@
                     "indicatorHeight": {
                       "type": "integer"
                     },
+                    "autoplay": {
```

Comment:
default is false according to your code. Default=false is what we want, just just change the schema here.

Reply:
@vusters Can you check the new changes? I updated the two things (min value and default false changes) which you mentioned.